source/frontend/src/features/properties/map/compensation/CompensationRequisitionYupSchema.ts

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+/* eslint-disable no-template-curly-in-string */
+
+export const CompensationRequisitionYupSchema = yup.object().shape({
+  agreementDate: yup.string(),
+  expropriationNoticeServedDateTime: yup.string(),
+  expropriationVestingDateTime: yup.string(),
+  generationDatetTime: yup.string(),
+  status: yup.string(),
```

Comment:
not sure if this is an issue, but saving a compensation requisition with no status selected (neither final nor draft) results in a value of final being saved.

Reply:
marking the status as required.